src/chapters/data-types/exercises.rst

```diff
@@ -0,0 +1,64 @@
+Exercises: Data Types
+=====================
+
+To get started, open up Visual Studio and create a new console program by clicking *New* and selecting *Console Application* under *.Net Core App*.
```

Comment:
Can you make this more generic and/or add a todo to point to the part in the setup instructions to create a new program? im worried that mac-specific instructions won't translate to the windows setup

Reply:
Done! Thanks!